ops/testing.py

```diff
@@ -2105,6 +2141,8 @@ def __init__(self, unit_name: str, meta: charm.CharmMeta, config: '_RawConfig'):
         self._networks: Dict[Tuple[Optional[str], Optional[int]], _NetworkDict] = {}
         self._reboot_count = 0
         self._running_action: Optional[_RunningAction] = None
+        # for `Model.get_cloud_spec`, initialized to None
```

Comment:
Nit (full sentence including punctuation, UK spelling):

```suggestion
        # For `Model.get_cloud_spec`, initialised to None.
```